### PR TITLE
New services, update readme

### LINKED ISSUE #64: Backend: Scripts for nuking DB

### PR DESCRIPTION
**Pull request by Christdej**
Closes #64

### CODE REVIEW

File: backend/api/api.csproj [hunk 1]
```diff
@@ -5,6 +5,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Identity.Web" Version="1.0.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.8" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.8">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
```
[the remark this answers is not in the document]
Comment (Christdej) [reply backend/api/api.csproj:11]: This is needed for entity framework migrations

File: README.md [hunk 2]
```diff
@@ -5,3 +5,16 @@
...
+and that you have you database connectionstring set in appsettings.json:
+"Database": {
+    "ConnectionString": ""
+  }
+
+create initiall migration: dotnet ef migrations add InitialCreate
+
+delete database: dotnet ef database drop
+
+apply migrations: dotnet ef database update
```
[the remark this answers is not in the document]
Comment (Christdej) [reply README.md:20]: Done

File: README.md [hunk 3]
```diff
@@ -5,3 +5,16 @@
 We are using a Entity Framework SQL database for storing our data.
 The environment variable "Database__ConnectionString" can be a ADO.NET connection string to an existing
 database. If empty we use an InMemory database which is initialized with dummy data.
+
+Dotnet EF migrations:
+Make sure you have dotnet-ef installed: dotnet tool install --global dotnet-ef
+and that you have you database connectionstring set in appsettings.json:
+"Database": {
+    "ConnectionString": ""
+  }
```
[the remark this answers is not in the document]
Comment (Christdej) [reply README.md:14]: Done

File: README.md [hunk 4]
```diff
@@ -5,3 +5,16 @@
 We are using a Entity Framework SQL database for storing our data.
 The environment variable "Database__ConnectionString" can be a ADO.NET connection string to an existing
 database. If empty we use an InMemory database which is initialized with dummy data.
+
+Dotnet EF migrations:
```
[the remark this answers is not in the document]
Comment (Christdej) [reply README.md:9]: Done

**Review by ReedOnly — APPROVED** [review]

**Pull request closed without merging**